prow/scripts/cluster-integration/update-kyma-gke-integration.sh

```diff
@@ -0,0 +1,278 @@
+    KYMA_INSTALLER_IMAGE="${DOCKER_PUSH_REPOSITORY}${DOCKER_PUSH_DIRECTORY}/gke-integration/${REPO_OWNER}/${REPO_NAME}:COMMIT-${COMMIT_ID}"
+    export KYMA_INSTALLER_IMAGE
+fi
+
+
+### Cluster name must be less than 40 characters!
+export CLUSTER_NAME="${COMMON_NAME}"
+
+export GCLOUD_NETWORK_NAME="${COMMON_NAME_PREFIX}-net"
+export GCLOUD_SUBNET_NAME="${COMMON_NAME_PREFIX}-subnet"
+
+### For provision-gke-cluster.sh
+export GCLOUD_PROJECT_NAME="${CLOUDSDK_CORE_PROJECT}"
+export GCLOUD_COMPUTE_ZONE="${CLOUDSDK_COMPUTE_ZONE}"
+
+#Local variables
+DNS_SUBDOMAIN="${COMMON_NAME}"
+KYMA_SCRIPTS_DIR="${KYMA_SOURCES_DIR}/installation/scripts"
+KYMA_RESOURCES_DIR="${KYMA_SOURCES_DIR}/installation/resources"
+
+INSTALLER_YAML="${KYMA_RESOURCES_DIR}/installer.yaml"
+INSTALLER_CONFIG="${KYMA_RESOURCES_DIR}/installer-config-cluster.yaml.tpl"
+INSTALLER_CR="${KYMA_RESOURCES_DIR}/installer-cr-cluster.yaml.tpl"
+PROMTAIL_CONFIG_NAME=promtail-k8s-1-14.yaml
+
+#Used to detect errors for logging purposes
+ERROR_LOGGING_GUARD="true"
+
+shout "Authenticate"
+date
+init
+DNS_DOMAIN="$(gcloud dns managed-zones describe "${CLOUDSDK_DNS_ZONE_NAME}" --format="value(dnsName)")"
+
+if [[ "$BUILD_TYPE" != "release" ]]; then
+    shout "Build Kyma-Installer Docker image"
+    date
+    CLEANUP_DOCKER_IMAGE="true"
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-image.sh"
+fi
+
+NETWORK_EXISTS=$("${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/network-exists.sh")
+if [ "$NETWORK_EXISTS" -gt 0 ]; then
+    shout "Create ${GCLOUD_NETWORK_NAME} network with ${GCLOUD_SUBNET_NAME} subnet"
+    date
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-network-with-subnet.sh"
+else
+    shout "Network ${GCLOUD_NETWORK_NAME} exists"
+fi
+
+
+shout "Provision cluster: \"${CLUSTER_NAME}\""
+date
+export GCLOUD_SERVICE_KEY_PATH="${GOOGLE_APPLICATION_CREDENTIALS}"
+if [ -z "$MACHINE_TYPE" ]; then
+      export MACHINE_TYPE="${DEFAULT_MACHINE_TYPE}"
+fi
+if [ -z "${CLUSTER_VERSION}" ]; then
+      export CLUSTER_VERSION="${DEFAULT_CLUSTER_VERSION}"
+fi
+CLEANUP_CLUSTER="true"
+"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/provision-gke-cluster.sh"
+
+
+shout "Install Tiller"
+date
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value account)"
+"${KYMA_SCRIPTS_DIR}"/install-tiller.sh
+
+shout "Apply Kyma config"
+date
+
+if [[ "$BUILD_TYPE" == "release" ]]; then
+    echo "Use released artifacts"
+    gsutil cp "${KYMA_ARTIFACTS_BUCKET}/${RELEASE_VERSION}/kyma-config-cluster.yaml" /tmp/kyma-gke-integration/downloaded-config.yaml
+    gsutil cp "${KYMA_ARTIFACTS_BUCKET}/${RELEASE_VERSION}/kyma-installer-cluster.yaml" /tmp/kyma-gke-integration/downloaded-installer.yaml
+
+    kubectl apply -f /tmp/kyma-gke-integration/downloaded-installer.yaml
+
+    sed -e "s/__DOMAIN__/${DOMAIN}/g" /tmp/kyma-gke-integration/downloaded-config.yaml \
```

Comment:
Is `__DOMAIN__` needed for `xip`? According to documentation, it is not https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/04-03-cluster-installation.md#using-the-latest-github-release